logdevice/ops/ldquery/tables/LogTreeBase.cpp

```diff
@@ -0,0 +1,81 @@
+#include "LogTreeBase.h"
+#include <folly/Conv.h>
+#include <folly/json.h>
+#include "../Utils.h"
+#include "logdevice/common/configuration/ReplicationProperty.h"
+
+namespace facebook::logdevice::ldquery::tables {
+
+TableColumns LogTreeBase::getAttributeColumns() {
+  return {
+      {"name", DataType::TEXT, "Name of the log group or the directory."},
+      {"logid_lo", DataType::LOGID,
+       "Defines the lower bound (inclusive) of the range of log ids in this "
+       "log group."},
+      {"logid_hi",
+       DataType::LOGID,
+       "Defines the upper bound (inclusive) of the range of log ids in this "
+       "log group."},
```

Comment:
These columns do not make sense for directories (at least with current implementation), what do you think about moving them into `LogGroups` class? Basically you can override `getAttributeColumns` in `LogGroups`, call `LogTreeBase::getAttributeColumns()` and add 2 more columns specific for this table.